### ISSUE TITLE
WG tunnel can't transfer data if we set excluded apps number more than 14, like the excluded apps number is 15, 16, etc.

### ISSUE THREAD

**Issue by egg1234** [wgtunnel/wgtunnel#247]
WG tunnel can't transfer data if we set excluded apps number more than 14, like the excluded apps number is 15, 16, 17, etc.

But WG tunnel will notify connected even the excluded apps number more than 14.

Thanks

**Comment by zaneschepke** [comment]
Hello! Very strange! I'll look into this.

**Comment by zaneschepke** [comment]
I think this one is fixed. Marking closed for now.

**Comment by egg1234** [comment]
Yes, I tested playstore version 3.9.4 and it seems fixed this issue.

I set excluded 20 apps and it work fine.

Thank you so much!
